src/MapfileStyleParser.ts

```diff
@@ -605,14 +615,16 @@ export class MapfileStyleParser implements StyleParser {
   getFillSymbolizerFromMapfileStyle(mapfileStyle: MapfileStyle): FillSymbolizer {
     const fillSymbolizer = { kind: 'Fill' } as FillSymbolizer;
 
-    if (!mapfileStyle.color && !mapfileStyle.symbol) {
+    if (!(mapfileStyle.color || mapfileStyle.outlinecolor) && !mapfileStyle.symbol) {
```

Comment:
I might not be up to speed with the latest geostyler style, but what about polygons with only a `outlinecolor `tag?

Reply:
This change takes into account polygons that are only defined by an outline color. Previously, these were provided with the property `visibility: false`, because the color attribute was missing. This is no longer the case and the style for the outline is applied.